README.md

```diff
@@ -21,78 +21,95 @@ It derives it's inspiration from the well known Java mocking framework [Mockito]
 ApexMocks allows you to write tests to both verify behaviour and stub dependencies.
 
 An assumption is made that you are using some form of [Dependency Injection](http://en.wikipedia.org/wiki/Dependency_injection) - for example passing dependencies via a constructor:
-
-	public MyClass(ClassA.IClassA dependencyA, ClassB.IClassB dependencyB)
-
+```Java
+public MyClass(ClassA.IClassA dependencyA, ClassB.IClassB dependencyB)
+```
 This allows you to pass mock implementations of dependencies A and B when you want to unit test MyClass.
 
 Lets assume we've written our own list interface fflib_MyList.IList that we want to either verify or stub:
-
-	public class fflib_MyList implements IList
```

Comment:
Can you revert all of the whitespace changes in this PR? It looks like your IDE is trying to replace tabs with spaces -- and it looks like most of the fflib-apex-mocks source uses tabs instead of spaces. We need to retain consistency with the established precedent in this project.

Reply:
Guess it depends.  Were you thinking I keep my setReadOnlyField but replace it with the internals of your forceFieldWrite?  If so, I can do the reverts also. Please let me know. Thanks.

Reply:
Done.